pkgs/tools/misc/slop/default.nix

```diff
@@ -25,6 +26,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LdBQxw8K8WWSfm4E2QpK4GYTuYvI+FX5gLOouVFSU/U=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "workaround-for-missing-glew-shared-library-at-link-time.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/naelstrof/slop/pull/135.patch";
```

Comment:
PRs can be updated over time, just point directly to the commit: https://github.com/naelstrof/slop/commit/811b7e44648b9dd6c1da1554e70298cf4157e5fe.patch

And better add a comment containing the PR title and link.